src/pages/devops/terraform/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Terraform
+---
+## Terraform
+
+Terraform is an infrastructure as code software by HashiCorp. It allows users to define a datacenter
+infrastructure in a high-level configuration language, from which it can create an execution plan to
+build the infrastructure in a service provider such as AWS.
```

Comment:
Dear @chankh 
I believe that the wikipedia definition is indeed a good one, but maybe it would be best to include the entire paragraph.

```Terraform is an infrastructure as code software by HashiCorp. It allows users to define a datacenter infrastructure in a high-level configuration language, from which it can create an execution plan to build the infrastructure such as OpenStack or in a service provider such as IBM Cloud (formerly Bluemix), AWS, Microsoft Azure or Google Cloud Platform. Infrastructure is defined in a HCL Terraform syntax or JSON format.```